ui/application-window.ui

```diff
@@ -31,7 +31,7 @@ Author: Philipp Wolfer <ph.wolfer@gmail.com>
     <property name="visible">True</property>
     <property name="can_focus">False</property>
     <property name="tooltip_text" translatable="yes">Start recording</property>
-    <property name="icon_name">media-playback-start-symbolic</property>
+    <property name="icon_name">media-record</property>
```

Comment:
I am not so keen on the look of this, can we change it to media-record-symbolic? Better fits the layout I think.

Reply:
Sure it looks good too me.